wfe/web-front-end.go

```diff
@@ -252,7 +252,7 @@ func (wfe *WebFrontEndImpl) NewRegistration(response http.ResponseWriter, reques
 		return
 	}
 
-	regURL := wfe.RegBase + string(reg.ID)
+	regURL := fmt.Sprintf("%s%d", wfe.RegBase, reg.ID)
```

Comment:
It did not break a test. I think we need to add two tests:

client integration testing: https://github.com/letsencrypt/boulder/issues/210
unittest: https://github.com/letsencrypt/boulder/issues/211